services/src/main/resources/scripts/authenticator-template.js

```diff
@@ -0,0 +1,26 @@
+// import enum for error lookup
```

Comment:
This is the default authenticator template that should be shown in the code editor for a quick start.